src/app/resources/iis_scripts/admin_api_util.ps1

```diff
@@ -0,0 +1,120 @@
+param(
+    [Parameter(Mandatory=$true)]
+    [ValidateSet('install', 'permission-check')]

```

Comment:
_permission-check_ is unused.

Reply:
What I meant was that it's not used in the script itself.

Reply:
Yeah, permission-check is just there as a enum value currently mean "not install". We might find that we need to do a different thing in the future.